app/src/main/java/com/seafile/seadroid2/cameraupload/CameraSyncAdapter.java

```diff
@@ -302,7 +306,7 @@ public void onPerformSync(android.accounts.Account account,
 
             // wait for TransferService to connect
             // Log.d(DEBUG_TAG, "waiting for transfer service");
-            int timeout = 1000; // wait up to a second
+            int timeout = 10000; // wait up to a second
```

Comment:
10 seconds, comment is wrong.

Reply:
Sorry..., I forget to modify the comments. I will fix it in the next version.